packages/lookit-api/rollup.config.dev.mjs

```diff
@@ -0,0 +1,8 @@
+import { makeRollupConfig } from "@jspsych/config/rollup";
+
+import { makeDevConfig } from "../../rollup-dev.mjs";
+
+const rollupConfig = makeRollupConfig("lookitInitJsPsych");
```

Comment:
Should this be `makeRollupConfig("lookitApi")`?

Reply:
Yes, it should. Thank you.